x-pack/plugins/security_solution/public/management/cypress/support/data_loaders.ts

```diff
@@ -220,5 +221,69 @@ export const dataLoadersForRealEndpoints = (
       const { kbnClient } = await stackServicesPromise;
       return destroyEndpointHost(kbnClient, createdHost).then(() => null);
     },
+
+    createFileOnEndpoint: async ({
```

Comment:
If you end up keeping these tasks, then I suggest you rename these and replace the word `Endpoint` with `Host`, in order to not cause confusion with our Endpoint executable.

But also, the more important comment here is that these do not apply to the Mocked data suite, so I suggest that you only register them for the `endpoint` test suite.